tests/memoryview/memview_assignments.pyx

```diff
@@ -4,3 +4,11 @@
 cdef double[::1] contig
 # see if we can assign a strided value to a contiguous one
 contig[:] = contig[::2]
+
+# see if we can assign memoryview across aliased types
+ctypedef double T
+cdef:
+    double[:] x
+    T[:] y
+x = y
```

Comment:
I think it would be better to use the test case in the original issue. I am just not so sure about having this cyclic assignment